sdk/ai/azure-ai-assistants/samples/async_samples/sample_assistant_functions.py

```diff
@@ -0,0 +1,122 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to use assistants operations with user function call from
+    the Azure Assistants service using a synchronous client.
+
+    See package documentation:
+    https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-assistants/README.md#key-concepts
+
+USAGE:
+    python sample_assistant_functions.py
+
+    Set these two environment variables before running the sample:
+    1) AZUREAI_ENDPOINT_URL - Your endpoint URL, in the form 
+        https://<your-deployment-name>.<your-azure-region>.models.ai.azure.com
+        where `your-deployment-name` is your unique AI Model deployment name, and
+        `your-azure-region` is the Azure region where your model is deployed.
+    2) AZUREAI_ENDPOINT_KEY - Your model key (a 32-character string). Keep it secret.
+"""
+import asyncio
+from opentelemetry import trace
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+
+from azure.ai.assistants.aio import AssistantsClient
+from azure.ai.assistants.models import FunctionTool
+from azure.core.credentials import AzureKeyCredential
+from user_functions import user_functions
```

Comment:
I want to reuse user_functions in the samples folder, please tell me how.  I can't get import statement to work.

Reply:
from ..user_functions.py import user_functions, would that work?

Reply:
Doesn't work.   I get: 
ImportError: attempted relative import with no known parent package
My guess is, I need to create samples/__init__.py to include use_functions in __all__ .   But I don't like to create such file.    Perhaps we should have duplicated copy of user_function.   Or move the user function into sample_assistant_function.py and sample_assistant_function_async.py